content/docs/self-hosted-runners.md

```diff
@@ -437,17 +437,23 @@ provisioned through environment variables instead of files.
 
 The `cml runner` command can also be used to manually set up a local machine,
 on-premise GPU cluster, or any other cloud compute resource as a self-hosted
-runner. Simply [install CML](/doc/install) and then run:
+runner. Instead of having a `launch-runner` job (as in the YAML above), simply
+[install CML](/doc/install) on the local machine and then run:
 
 ```cli
 $ cml runner launch \
   --repo="$REPOSITORY_URL" \
   --token="$PERSONAL_ACCESS_TOKEN" \
-  --labels="local,runner" \
+  --labels="cml-gpu" \
   --idle-timeout=180
 ```
 
 The machine will listen for jobs from your repository and execute them locally.
+The label(s) may be anything, but at least one of them must match the one
+configured in the `runs_on` of the YAML.
```

Comment:
It may be better to keep more than one label in the sample above if we then say "at least one of them".

And where is `runs_on` shown? I don't see it in the code samples of this page.

Reply:
In the example YAML, look for:

```yaml
train-and-report:
    needs: launch-runner
    runs-on: [self-hosted, cml-gpu]
```

Reply:
as per https://github.com/iterative/cml.dev/pull/417/files#r1046427536, moved to follow-up #412